CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change history for ui-users
 
+## [7.0.0] (IN PROGRESS)
```

Comment:
We do not know if there will be major changes yet, so probably it should be `v6.1.0`, right?

Reply:
@zburke @mkuklis I use next unreleased version from jira. Could you please clarify what version we should use?

Reply:
@Dmitriy-Litvinenko I think `6.1.0` should be good here.

Reply:
True, we don't know. This is just the CHANGELOG so NBD either way. Arguably, the person who made the last release should have then pushed another PR to update `#master` to the next expected version, and adjusted the IN PROGRESS version here. I'll go find that person and remind them about their responsibilities. Oh, wait ... 😅

Reply:
Thank you, i change fix version to 6.1.0